pydocx/openxml/packaging/main_document_part.py

```diff
@@ -0,0 +1,65 @@
+# coding: utf-8
+from __future__ import (
+    absolute_import,
+    print_function,
+    unicode_literals,
+)
+
+from pydocx.openxml.packaging.font_table_part import FontTablePart
+from pydocx.openxml.packaging.footnotes_part import FootnotesPart
+from pydocx.openxml.packaging.image_part import ImagePart
+from pydocx.openxml.packaging.numbering_definitions_part import NumberingDefinitionsPart  # noqa
+from pydocx.openxml.packaging.open_xml_part import OpenXmlPart
+from pydocx.openxml.packaging.style_definitions_part import StyleDefinitionsPart  # noqa
+
+
+class MainDocumentPart(OpenXmlPart):
+    '''
+    Represents the actual document XML tree within a Word document container.
+    This OpenXmlPart exposes several child parts for styles, numbering, fonts
+    and images.
+
+    See also: http://msdn.microsoft.com/en-us/library/documentformat.openxml.packaging.maindocumentpart%28v=office.14%29.aspx  # noqa
+    '''
+
+    relationship_type = '/'.join([
+        'http://schemas.openxmlformats.org',
```

Comment:
No, but good for checking. http://en.wikipedia.org/wiki/XML_namespace#Namespace_names

XML just uses URIs for namespacing, because they are guaranteed to be globally unique. 

> In general, however, users should assume that the namespace URI is simply a name, not the address of a document on the Web.